install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -46,6 +46,11 @@ initContainers:
     privileged: true
     {{- end }}
   restartPolicy: Always
+  env:
+  {{- if contains "*" (annotation .ObjectMeta `traffic.sidecar.istio.io/includeInboundPorts` "DEFAULT_AS_NOT_STAR") }}
```

Comment:
Doesn't "" work (instead of DEFAULT_AS...) ?

Reply:
Empty string  works. But `DEFAULT_AS` is more clear.

Reply:
Use `ISTIO_META_INBOUND_CAPTURE_PORTS`
It could be "*" or "," joined ports

Reply:
I don't think I've seen this pattern anywhere else in the code - better to avoid confusing people.

Reply:
Sure. Will follow the convention: use empty string literal

Reply:
fixed